docs/content/develop/properties.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "Define properties"
+description = ""
+weight = 45
+draft = false
+bref = ""
+toc = true
+menu = "develop"
+tags = [ "develop" ]
++++
+
+Property definitions provide a standard, streamlined way to maintain state within your component, receive external input (via attributes), initialize values, trigger handler functions, and more.
+
+## Introduction
+
+To introduce the topic, consider a "Counter" component that counts how many times something has happened.  Such a component could define a counter like this.
+
+```javascript
+static get properties() {
+  return {
+    count: {
+      type: Number,
+      default: 0
+    }
+  };
+}
+```
+
+Defining `count` in this way brightens your day in the following ways:
+
+ - **A property**: The property's name is `count`, so `this.count` is created and initialized to the default value, the number `0`.  
+ - **An attribute**: An attribute, `pfe-c-count` is also created, and will be kept in sync with `this.count`.  The `pfe-c-` prefix is explained [below](#prefix).
+
+Below, all the options for property definitions are explained.
+
+## Property / attribute binding
+
+As mentioned in the introduction, defining a property also creates an attribute, and the two values will be kept in sync.  For properties with the String type, this is straightforward.  Attribute values are always strings, and so logic is needed to 
+
+### Handling `null`
+
+If a property is assigned to null, the associated attribute will be removed.
+
+Example: `this.count = null` will result in `this.removeAttribute("pfe-c-count")`.
+
+## Type
+
+Three property types are supported: String, Number, and Boolean.
+
+### String
+
+```
+name: {
+  type: String
+}
+```
+
+### Boolean
+
```

Comment:
```suggestion
Though all attribute values are technically strings, if you are assigning a `true/ false` paradigm or checking that a variable exists (implicit `true/false`, i.e., `hidden` or `required`), please assign that property a type of `Boolean`.  This will ensure the update and injection of that value is handled correctly by the base class.
```

Reply:
I left a note in your similar suggestion for Number. Looking forward to your thoughts.

Reply:
Added my thoughts to that thread, let me know.  I think the additional information is extremely helpful.